opencsp/app/sofast/test/test_SofastConfiguration.py

```diff
@@ -61,12 +61,14 @@ def _get_process_sofast_fixed():
         measurement = MeasurementSofastFixed.load_from_hdf(file_meas)
 
         # Instantiate SofastFixed class and load measurement data
-        sofast = ProcessSofastFixed(orientation, camera, fixed_pattern_dot_locs, facet_data)
+        sofast = ProcessSofastFixed(orientation, camera, fixed_pattern_dot_locs)
         sofast.load_measurement_data(measurement)
 
         # Process
         surface = Surface2DParabolic(initial_focal_lengths_xy=(150.0, 150), robust_least_squares=False, downsample=1)
```

Comment:
Any particular reasons for seeding with (150.0, 150)? If so, please add brief comment.

Would this test be more robust by generating random focal lengths?

Reply:
Commentes added